api/watch/funcs_test.go

```diff
@@ -29,19 +35,68 @@ func makeInvokeCh() chan error {
 	return ch
 }
 
+func updateConnectCA(t *testing.T, client *api.Client) {
+	t.Helper()
+
+	connect := client.Connect()
+	cfg, _, err := connect.CAGetConfig(nil)
+	require.NoError(t, err)
+	require.NotNil(t, cfg.Config)
+
+	// update the cert
+	// Create the private key we'll use for this CA cert.
+	pk, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Maybe you could just create some 10-year certs and commit them as test-fixtures like vault does: https://github.com/hashicorp/vault/tree/master/api/test-fixtures/keys

Reply:
hmm, that certainly would work. This is just to get those watches to trigger a second time anyways.

Reply:
FYI we already have certs in a test fixture committed in `test/ca` as matt said though this is just triggering rotation as part of the test so generating a new key seems reasonable. All the Connect tests generate keys live during each test case for example.